.travis.yml

```diff
@@ -42,16 +51,19 @@ jobs:
     script:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest ES_TEST=frontend
+
   - name: Reactive-sandbox test
     script:
     - make test CHARTS=reactive-sandbox
+
   - stage: deploy
     name: Deploy to GCS
     # Only deploy for pushes to master (and if manually invoked)
     if: type IN (push, api)
     script:
     # Decrypt credentials for GCS system account helmchart@es-repo.iam.gserviceaccount.com
-    - openssl aes-256-cbc -K $encrypted_f01ffbb90c44_key -iv $encrypted_f01ffbb90c44_iv -in resources/es-repo-7c1fefe17951.json.enc -out resources/es-repo-7c1fefe17951.json -d
+    - mkdir -p /tmp/resources
+    - openssl aes-256-cbc -K $encrypted_f01ffbb90c44_key -iv $encrypted_f01ffbb90c44_iv -in resources/es-repo-7c1fefe17951.json.enc -out /tmp/resources/es-repo-7c1fefe17951.json -d
```

Comment:
Curious about this change.  Is this just to keep source and generated artifacts separate?  Is there a need to put things in `/tmp` given that everything is going to get blown away anyway once the test runs to completion?  (I suspect I'm missing some subtlety...)

Reply:
It's so we don't accidentally upload the encrypted file to S3. By default the artifacts tool will upload anything generated inside the git repo. So this is just being "better safe than sorry".